src/text/skeleton.scss

```diff
@@ -3,23 +3,29 @@
 @import '../style/index';
 
 @mixin iui-text-skeleton {
-  display: inline-block;
+  width: fit-content;
   user-select: none;
   color: transparent;
   border-radius: $iui-border-radius;
+  cursor: progress;
 
   @include themed {
     background: linear-gradient(
-      270deg,
-      t(iui-color-background-3),
-      t(iui-color-background-3),
-      t(iui-color-background-2),
-      t(iui-color-background-3),
-      t(iui-color-background-3)
+      292deg,
+      t(iui-text-color-muted),
```

Comment:
Idk, for me previous one looked better 😀 Maybe we could use some color combination?

Reply:
I liked the appearance of the older one better as well, but it did not pass accessibility.

Reply:
1 idea I had was to use `currentColor` so the skeleton text inherits the color.  You could use skeleton text in a CTA button for example.  I think to do that I would need to use `::after` to display the skeleton though.

Reply:
![Screen Shot 2022-04-14 at 11 04 23 AM](https://user-images.githubusercontent.com/849817/163418698-7471de86-d73a-4420-85a9-91243b1c8d30.png)

Reply:
Aren't they too dark in the light theme?

Reply:
By using the text colors we know that we're passing contrast scores.  I know they are more prominent visually, but I think accessibility trumps visuals here.